### PR TITLE
Implemente API for getting a random quote. Close #87.

### LINKED ISSUE #87: API for getting random quote
This is essential for https://github.com/codingteam/horta-hell/issues/297

### PR DESCRIPTION
**Pull request by rexim**

### CODE REVIEW

File: scalajvm/app/models/queries/QuoteQueries.scala [hunk 1]
```diff
@@ -73,6 +73,13 @@ case class QuoteQueries(implicit session: DBSession) {
     }.map(rs => Quote(rs)).first().apply()
   }
 
+  def getRandomQuote: Option[Quote] = {
+    val q = Quote.syntax("q")
+    withSQL {
+      select(q.*).from(Quote as q).append(sqls"ORDER BY RANDOM()").limit(1)
```
Comment (ForNeVeR) [inline scalajvm/app/models/queries/QuoteQueries.scala]: Are you sure we will be satisfied by performance of this query? There are known problems with `order by random`.

Comment (rexim) [inline scalajvm/app/models/queries/QuoteQueries.scala]: Hm... That's a good point, thank you. I'll think about it.

Comment (ForNeVeR) [inline scalajvm/app/models/queries/QuoteQueries.scala]: The most portable idea was to get the row count first and then either calculate the random value in the application code or to do something like `select * from q offset random() * @count limit 1`.

**Comment by ForNeVeR** [comment]
Okay!

**Pull request merged**